jena-fuseki1/src/test/java/org/apache/jena/fuseki/ServerTest.java

```diff
@@ -20,7 +20,7 @@
 
 import java.io.IOException ;
 import java.net.ServerSocket ;
-import java.util.Arrays ;
+import java.util.concurrent.atomic.AtomicInteger;
```

Comment:
I have no problem switching it back. I liked the clarity and concision of `getAndIncrement()` and `decrementAndGet()`, but maybe that's just me.